src/Aspire.ProjectTemplates/templates/aspire-empty/AspireApplication.1.ServiceDefaults/Extensions.cs

```diff
@@ -101,17 +101,22 @@ public static IHostApplicationBuilder AddDefaultHealthChecks(this IHostApplicati
 
     public static WebApplication MapDefaultEndpoints(this WebApplication app)
     {
-        // Uncomment the following line to enable the Prometheus endpoint (requires the OpenTelemetry.Exporter.Prometheus.AspNetCore package)
-        // app.MapPrometheusScrapingEndpoint();
+        // Adding health checks endpoints to applications in non-development environments has security implications.
+        // See https://aka.ms/dotnet/aspire/healthchecks for details before enabling these endpoints in non-development environments.
+        if (app.Environment.IsDevelopment())
```

Comment:
Was there anything relying on these endpoints? For example, does ACA/k8s query them at all? If not, how does ACA know if the app is health or not?

Reply:
Nothing in our E2E relies on these today.